lib/reply.js

```diff
@@ -515,7 +513,7 @@ function notFound (reply) {
 
 function noop () {}
 
-function getHeaderProper (reply, key) {
+function getHeader (reply, key) {
```

Comment:
I think that this code could directly be written inside `Reply.prototype.getHeader`.

Reply:
yes that way it's better, thanks for the hints; code just updated